src/pages/layout.js

```diff
@@ -10,76 +10,189 @@ import i18n from '@dhis2/d2-i18n'
 
 import { D2Shim } from '../utils/D2Shim'
 import layoutStyles from '../styles/Layout.module.css'
+import api from '../utils/api'
+import {
+    DATASET_SETTINGS,
+    GENERAL_SETTINGS,
+    NAMESPACE,
+    PROGRAM_SETTINGS,
+} from '../constants/data-store'
+import { androidSettingsDefault } from '../constants/android-settings'
+import { SpecificSettingsDefault } from '../constants/program-settings'
+import { DataSetSettingsDefault } from '../constants/data-set-settings'
```

Comment:
Could you make these objects lowerCase, like `androidSettingsDefault`? This is for convention and consistency :)

Reply:
hehehe okey dokey, thanks for the advice :)